src/components/Table/GridTable.tsx

```diff
@@ -220,13 +225,13 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   const columnSizes = useSetupColumnSizes(style, columns, resizeTarget ?? resizeRef);
 
   // Make a single copy of our current collapsed state, so we'll have a single observer.
-  const collapsedIds = useComputed(() => rowState.collapsedIds, [rowState]);
+  const collapsedIds = useComputed(() => tableState.collapsedIds, [tableState]);
+  const sortState = useComputed(() => tableState.sortState, [tableState]);
```

Comment:
Question: I'm kinda surprised, how does `sortState.current` being changed get this `sortState` to change & trigger a re-render?

Is it because we do:

```
      this.sort = newState ?? {};
```

If so, I'm just a little surprised... B/c if someone did:

```
    this.sort.current = ...new current...;
```

Then AFAIU this line would definitely not trigger/re-render, b/c the only observer we're triggering is "read the `sortState`" value.

But I guess mobx could see the `this.sort = ...new value...` as fundamentally changing the proxy identity I guess? That works out.

FWIW there is also a mobx utility method, `toJS`, that is basically a deep copy, similar to the `{ ...copy }` trick to trigger a re-render whenever a field on the object changes, b/c we've "read each field" while doing the `...`, but `toJS` will do it deeply.

I don't think we need it here, but just calling out that something like:

```
useComputed(() => toJS(tableState.sortState))
```

Would be safer if deep keys were involved.

I kind wonder if maybe `useComputed` should examine its return value, and if its a proxy, just auto-toJS it... :thinking: 

Anyway, just thoughts.

Reply:
Yea, thats a good point. We did get lucky here because we are _always_ updating/replacing the whole `this.sort` object and not just the inner values. You are correct that just changing `current` would not be reactive, though probably should be.
I gave a test with the `toJS` recommendation, and that works as expected when just changing `current`, or an inner value.